prime-router/src/main/kotlin/metadata/LivdMappers.kt

```diff
@@ -84,9 +84,10 @@ class LIVDLookupMapper : Mapper {
         val filters = lookupTable.FilterBuilder()
         // get the test performed code for additional filtering of the test information in case we are
         // dealing with tests that check for more than one type of disease, for example COVID + influenza
-        values.firstOrNull { it.element.name == ElementNames.TEST_PERFORMED_CODE.elementName }?.value?.also {
-            filters.notEqualsIgnoreCase(LivdTableColumns.TEST_PERFORMED_CODE.colName, testProcessingModeCode)
-        }
+        values.firstOrNull { it.element.name == ElementNames.TEST_PERFORMED_CODE.elementName }?.value
+            ?.also { testPerformedCode ->
+                filters.equalsIgnoreCase(LivdTableColumns.TEST_PERFORMED_CODE.colName, testPerformedCode)
```

Comment:
Note the change from notEqualsIgnoreCase to equalsIgnoreCase, and the use of the proper testPerformedCode value